cluster/manifests/spot-node-rescheduler/rbac.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if eq .ConfigItems.spot_node_rescheduler "true" }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spot-node-rescheduler
+  namespace: kube-system
+  annotations:
+    iam.amazonaws.com/role: "{{ .Cluster.LocalID }}-spot-node-rescheduler"
```

Comment:
There is no AWS role needed for the spot-node-rescheduler so we can drop this annotation.